test/aria/widgets/wai/multiselect/MultiSelectJawsTest.js

```diff
@@ -0,0 +1,79 @@
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Aria.classDefinition({
+    $classpath : "test.aria.widgets.wai.multiselect.MultiSelectJawsTest",
+    $extends : "aria.jsunit.JawsTestCase",
+    $constructor : function () {
+        this.$JawsTestCase.constructor.call(this);
+        this.setTestEnv({
+            template : "test.aria.widgets.wai.multiselect.MultiSelectTpl"
+        });
+    },
+    $prototype : {
+
+        /**
+         * This method is always the first entry point to a template test Start the test by focusing the first field
+         */
+        runTemplateTest : function () {
+            var checkedRegExp = /not checked\nchecked/g;
+            var notCheckedRegExp = /checked\nnot checked/g;
+
+            this.synEvent.execute([
+                ["click", this.getElementById("tf")],
+                ["pause", 2000],
+                ["type", null, "[down][down]"],
+                ["pause", 1000],
+                ["type", null, "[<shift>][F10][>shift<]"],
+                ["pause", 1000],
+                ["type", null, "[space]"],
+                ["pause", 500],
+                ["type", null, "[down]"],
+                ["pause", 500],
+                ["type", null, "[space]"],
+                ["pause", 500],
+                ["type", null, "[down][down]"],
+                ["pause", 500],
+                ["type", null, "[space]"],
+                ["pause", 500],
+                ["type", null, "[down]"],
+                ["pause", 500],
+                ["type", null, "[down]"],
+                ["pause", 500],
+                ["type", null, "[space]"],
+                ["pause", 500],
+                ["type", null, "[escape]"],
+                ["pause", 500],
+                ["type", null, "[down]"],
+                ["pause", 500],
+                ["type", null, "[space]"],
+                ["pause", 1000]
+            ], {
+                fn: function () {
+                    this.assertJawsHistoryEquals(
+                        "Here is the default Multi-Select: Edit\nType in text.\nMy Multi-select:\nEdit\nAir Canada check box checked\nAir France\ncheck box not checked\nAir France check box checked\nAir New Zealand\ncheck box not checked\nBritish Airways\ncheck box not checked\nBritish Airways check box checked\nDelta Airlines\ncheck box not checked Unavailable\nFinnair\ncheck box not checked Unavailable\nedit\nAir Canada,Air France,British Airways\nType in text.\nPress space to open the selection list\nbutton menu collapsed\nAir Canada check box checked",
```

Comment:
It seems that JAWS randomly inserts a new line before saying `check box checked` or `check box not checked`.
I think it would be nice to add a filter to remove the new line before `check box` to make the output more consistent.